src/similar-npm-packages.ts

```diff
@@ -501,6 +521,16 @@ export const categories: Categories = {
     ],
   },
 
+  "react-slider": {
+    name: "Slider components for React",
```

Comment:
I'm not sure that it's a good idea since they are actually sliders. Call them "range input" means that they can't have one handle, because "range" apriori means selection of 2 limits.